packages/backend/src/models/AnalyticsModel.ts

```diff
@@ -74,13 +74,21 @@ export class AnalyticsModel {
 
     async addChartViewEvent(
         chartUuid: string,
-        userUuid: string,
+        userUuid: string | null,
     ): Promise<void> {
         await this.database.transaction(async (trx) => {
-            await trx(AnalyticsChartViewsTableName).insert({
+            const payload: {
+                chart_uuid: string;
+                user_uuid?: string;
+            } = {
                 chart_uuid: chartUuid,
-                user_uuid: userUuid,
-            });
+            };
+
+            if (userUuid) {
+                payload.user_uuid = userUuid;
+            }
+
+            await trx(AnalyticsChartViewsTableName).insert(payload);
```

Comment:
We shouldn't need all of this since the insert should support "userUuid: null". 

The problem is that the table types are wrong in `packages/backend/src/database/entities/analytics.ts` and `packages/backend/src/@types/knex-tables.d.ts`